be/src/runtime/tablets_channel.cpp

```diff
@@ -436,9 +436,10 @@ Status TabletsChannel::add_batch(const PTabletWriterAddBlockRequest& request,
         }
     }
 
-    auto get_send_data = [&]() { return vectorized::Block(request.block()); };
+    vectorized::Block send_data;
+    auto get_send_data = [&]() { return send_data.deserialize(request.block()); };
```

Comment:
should create a new block and then return it.
if the get_send_data is called multiple times then it is wrong.